### PR TITLE
Updated packaging

### PR DESCRIPTION
**Pull request by lslezak**
- regenerated *.spec file by `gem2rpm` (to use the new Ruby macros)
- fixed runtime dependencies (were in a wrong section in `gem2rpm.yml`)
- use the new ruby packaging (builds packages for both Ruby 2.1 and Ruby 2.2)
- 0.0.2

### CODE REVIEW

File: package/gem2rpm.yml [hunk 1]
```diff
@@ -47,7 +45,7 @@
 #   install -D -m 0644 %{S:1} %{buildroot}%{_bindir}/gem2rpm-opensuse
 # ## used by gem2rpm
 :testsuite_command: |-
-   (pushd %{buildroot}%{gem_base}/gems/%{mod_full_name} && rake check:ci)
+   (pushd %{buildroot}%{gem_base}/gems/%{mod_full_name} && rake --verbose --trace check:ci)
```
Comment (mvidner) [inline package/gem2rpm.yml]: `(pushd foo; bar)` is superfluous needless extra duplication ;-) Use `pushd foo; bar; popd` or `(cd foo; bar)`.

**Comment by mvidner** [comment]
LGTM.

File: package/gem2rpm.yml (hunk 1 above)
Comment (lslezak) [inline package/gem2rpm.yml]: I originally took the `gem2rpm.yml` template and just replaced `rake test` by `rake check:ci`, the template is to blame :wink:

**Comment by mvidner** [comment]
Looks Even Better Now! :+1:

**Pull request merged**